buildpipeline/Core-Setup-CrossBuild.json

```diff
@@ -61,32 +61,34 @@
       "displayName": "Start detached container",
```

Comment:
This is not directly related to your change, but I'm not seeing why we have 4 docker steps - start container, build in container, stop container, remove container.  Seems like this could be accomplished in one step.